doc/classes/BitMap.xml

```diff
@@ -71,6 +71,12 @@
 			<argument index="1" name="epsilon" type="float" default="2.0">
 			</argument>
 			<description>
+				Creates an [code]Array[/code] of polygons covering a rectangular portion of the bitmap. Each polygon is described as a [code]PoolVector2Array[/code] of its vertices.
+				To get polygons covering the whole bitmap, pass:
+				[codeblock]
+				Rect2(0, 0, get_size().x, get_size().y)
+				[/codeblock]
+				[code]epsilon[/code] controls how accurately the polygons cover the bitmap: a lower [code]epsilon[/code] corresponds to more points in the polygons.
```

Comment:
Maybe worth to note that this uses [marching squares](https://en.wikipedia.org/wiki/Marching_squares) as well.
```suggestion
				[code]epsilon[/code] controls how accurately the polygons cover the bitmap during marching squares algorithm: a lower [code]epsilon[/code] corresponds to more points in the polygons.
```
Or as a separate paragraph.

Reply:
It also uses [RDP](https://en.wikipedia.org/wiki/Ramer%E2%80%93Douglas%E2%80%93Peucker_algorithm) algorithm for reducing points.

Reply:
Not an expert here, but looking at the code and wikipedia, epsilon is a parameter of RDP, not marching squares.